src/Chapter2.hs

```diff
@@ -336,7 +336,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end l
+  | start < 0 || end < 0 || end < start = []
+  | otherwise = drop (start) (take (end+1) l)
```

Comment:
Great solution with all cases handled 👏🏼 

Also, some of the parenthesis are redundant, and operators are not separated by spaces:
```suggestion
  | otherwise = drop start (take (end + 1) l)
```